client/src/components/UnsavedChangesIndicator/UnsavedChangesIndicator.scss

```
@@ -0,0 +1,70 @@
+.unsaved-changes-indicator__container {
+  display: flex;
+}
+
+.unsaved-changes-indicator {
+  display: inline-flex;
+  gap: 8px;
+  padding: 7px 8px 0;
+  border-radius: 3px;
+  border: 1px solid;
+  font-size: 13px;
+  white-space: nowrap;
+  margin-left: 4px;
+  margin-right: 8px;
+
+  // These use bootstrap colors
+  &--notice {
+    // --bs-info-text-emphasis
+    color: rgb(16.4, 68.4, 90.4);
+    // --bs-info-border-subtle
+    border-color: rgb(169.4, 221.4, 243.4);
+    // --bs-info-bg-subtle
+    background-color: rgb(212.2, 238.2, 249.2);
+  }
+
+  &--warning {
+    // --bs-warning-text-emphasis
+    color: rgb(102, 77.2, 2.8);
+    // --bs-warning-border-subtle
+    border-color: rgb(255, 230.2, 155.8);
+    // --bs-warning-bg-subtle
+    background-color: rgb(255, 242.6, 205.4);
+  }
+
+  .font-icon {
+    display: inline-block;
+    font-size: 14px;
+  }
```

```suggestion
  .unsaved-changes-indicator__icon {
    display: inline-block;
    font-size: 14px;
  }
```
Should ideally use BEM instead of nesting.

---

Updated